### ISSUE TITLE
Accessibility -- Unable to interact with visual using keyboard only.

### ISSUE THREAD

**Issue by browntc7** [microsoft/PowerBI-visuals-ChicletSlicer#72]
Unable to interact with visual without using the mouse. 

No attempt to interact with the visual using the keyboard produced any result. Tried all keyboard shortcuts in the CTRL + ? menu. 

The plan is to use this visual in Power BI Embedded (not publish to web) on a public facing website, but this visual needs to be accessible to those who may only use keyboards.

**Comment by helen508light** [comment]
Thank you for your feedback.
We'll consider your suggestions in the next versions of the visual.

Please note that source code for the visual is public, feel free to fork and modify it for your needs.

**Comment by ktiris** [comment]
I am a data visualizer and would also like this enhancement addressed.  This is an outstanding feature that increases engagement.  However, it does not currently meet [Section 508](https://www.section508.gov/manage/laws-and-policies) of the US Code.  When our accessibility specialist tested our dashboard, it did not pass because individuals using screen readers would not be able to filter the dashboard.  Thank you so much!

**Comment by memmes** [comment]
The Chiclet Slicer is still inaccessible to non-mouse users and fails accessibility requirements (WCAG/Section 508), using version 1.6.3. Is there an update for when we might see accessibility enhancements available in production for this visual?

**Comment by helen508light** [comment]
@memmes @ktiris 
This enhancement is already in our backlog, but there is no ETA for now

**Comment by roys-msft** [comment]
Any updates on this? The slicer is still not meeting accessibility requirements. Any ETA on when this will be fixed?
